eng/service.proj

```diff
@@ -35,4 +35,13 @@
              SkipNonexistentTargets="true" />
   </Target>
 
+  <!--Run Aggregate Updates to Source -->
+  <Target Name="AggregateUpdatesToSource" AfterTargets="Build" Condition="'$(UpdateSource)' == 'true' or '$(AZURE_DEV_UPDATESOURCES)' == 'true'" >
```

Comment:
This won't work in VS would it?

Reply:
You would need to set the env variable before launching VS from the command line. Or have another way of setting this property from msbuild.

Reply:
Nobody opens the `eng/service.proj` in VS

Reply:
Ah yes that is fair. this should go into the common targets file.